### ISSUE TITLE
feat: min-icon-size

### ISSUE THREAD

**Issue by justchokingaround** [emersion/mako#464]
there is already a max-icon-size option, can we have a min-icon-size option please?

**Comment by emersion** [comment]
But this would scale up and make the icons blurry?

**Comment by justchokingaround** [comment]
yes, however it would make the notifications with icons look more consistent (in my case for song covers)

**Comment by Jark5455** [comment]
vector graphics icons can benefit from this and won't get blurry.
